dot/core/service_integration_test.go

```diff
@@ -462,8 +528,9 @@ func TestMaintainTransactionPool_BlockWithExtrinsics(t *testing.T) {
 		}
 		res = append(res, tx)
 	}
-	require.Equal(t, 1, len(res))
-	require.Equal(t, res[0], txs[1])
+	// Extrinsic is removed. so empty res
+	require.Equal(t, 0, len(res))
+	require.Equal(t, res, []*transaction.ValidTransaction{})
```

Comment:
no need to compare length due to the 2nd assertion + expected argument is mean to go as 2nd argument, not 3rd.

```suggestion
	require.Equal(t, []*transaction.ValidTransaction{}, res)
```